### DISCUSSION TITLE
quarkus native dynatrace

### DISCUSSION THREAD

**Discussion by tbkahuna48** [quarkusio/quarkus#27188] [Q&A]
Is anyone aware of any work being done to have quarkus native provide the same level of support for dynatrace features as JVM versions? 
Large companies we are often stuck with certain monitoring solutions and requirements, for us it is dynatrace. Writing our own exporter to write to the gRPC dynatrace collector endpoint, in Azure cloud incurs additional Davis Data Unit charges for each span.
Creating a dynatrace injectable agent in the container's OS, would save those costs.

[a comment by an automated account is not shown]

**Comment by joergsesterhenn** [comment]
https://github.com/quarkusio/quarkus/issues/6149
